clients/drcachesim/tracer/raw2trace.h

```diff
@@ -1633,6 +1715,7 @@ class raw2trace_t : public trace_converter_t<raw2trace_t> {
                 const std::vector<std::ostream *> &out_files,
                 file_t encoding_file = INVALID_FILE, void *dcontext = nullptr,
                 unsigned int verbosity = 0, int worker_count = -1,
+                const std::string &syscall_pt_trace_dir = "",
```

Comment:
Seems like this should either be appended to avoid breaking existing uses or inserted at a meaningful place: it seems unusual to insert it here.  I would append it.  But see below on this model of raw2trace opening files and assuming they are local probably not being workable: we probably need a different interface here, passing in istreams.